exporters/otlp/common/src/main/java/io/opentelemetry/exporter/otlp/internal/WireFormat.java

```diff
@@ -48,13 +48,7 @@
   static final int MAX_VARINT64_SIZE = 10;
   static final int MAX_VARINT_SIZE = 10;
 
-  static final int WIRETYPE_VARINT = 0;
-  static final int WIRETYPE_FIXED64 = 1;
-  static final int WIRETYPE_LENGTH_DELIMITED = 2;
-  static final int WIRETYPE_FIXED32 = 5;
```

Comment:
We do have fixed32 usage in the logs proto in OTLP and we may have some coming in metrics in OTLP 0.10